ansible/roles/controller/tasks/deploy.yml

```diff
@@ -203,6 +203,9 @@
       "CONFIG_whisk_db_activationsFilterDdoc": "{{ db_whisk_activations_filter_ddoc | default() }}"
       "CONFIG_whisk_userEvents_enabled": "{{ user_events | default(false) | lower }}"
 
+      "CONFIG_whisk_credentials_controller_username": "{{ controller.username }}"
+      "CONFIG_whisk_credentials_controller_password": "{{ controller.password }}"
+
```

Comment:
If generate controller/invoker credentials to container's /conf/, `the Standalone Tests` run failed due to `lack /conf/ under the build machine`

On the other hand,  `couchdb credentials is passed via environment variable`, so controller/invoker credentials can be passed via environment variable as well.